src/dotnet/APIView/APIViewWeb/Managers/NotificationManager.cs

```diff
@@ -158,94 +164,84 @@ private string GetHtmlContent(CommentModel comment, ReviewModel review)
             return sb.ToString();
         }
 
-        private static string GetCommentTagPlainTextContent(CommentModel comment)
-        {
-            var sb = new StringBuilder();
-            sb.Append(GetCommentTagContentHeading(comment, false));
-            return sb.ToString();
-        }
-
-        private string GetPlainTextContent(CommentModel comment)
-        {
-            var sb = new StringBuilder();
-            sb.Append(GetContentHeading(comment, false));
-            sb.Append("\r\n");
-            sb.Append(CommentMarkdownExtensions.MarkdownAsPlainText(comment.Comment));
-            return sb.ToString();
-        }
-
-        private static string GetApproverReviewContentHeading(UserProfileModel user, bool includeHtml) =>
-            $"{(includeHtml ? $"<b>{user.UserName}</b>" : $"{user.UserName}")} requested you to review API.";
-
-        private static string GetCommentTagContentHeading(CommentModel comment, bool includeHtml) =>
-            $"{(includeHtml ? $"<b>{comment.Username}</b>" : $"{comment.Username}")} tagged you in a comment.";
-
         private static string GetContentHeading(CommentModel comment, bool includeHtml) =>
             $"{(includeHtml ? $"<b>{comment.Username}</b>" : $"{comment.Username}")} commented on this review.";
 
-        private async Task SendUserEmailsAsync(ReviewModel review, UserProfileModel user, string plainTextContent, string htmlContent)
+        private async Task SendUserEmailsAsync(ReviewModel review, UserProfileModel user, string htmlContent)
         {
-            var userBackup = new ClaimsPrincipal();
-            EmailAddress e;
-            if (!user.Email.IsNullOrEmpty())
-            {
-                e = new EmailAddress(user.Email, user.UserName);
-            }
-            else
+            // Always send email to a test address when test address is configured.
+            if (string.IsNullOrEmpty(user.Email))
             {
-                var backupEmail = GetUserEmail(userBackup);
-                if (!backupEmail.IsNullOrEmpty())
-                {
-                    e = new EmailAddress(backupEmail, user.UserName);
-                }
-                else
-                {
-                    return;
-                }
+                _telemetryClient.TrackTrace($"Email address is not available for user {user.UserName}. Not sending email.");
```

Comment:
Do we want to someone watch this? Do we have any upfront validation to ensure that if someone requests a review or mentioned someone that tells folks that they don't have an email configured? Or do we expect this to be an exceptional case because we are mapping to AAD accounts now?

Reply:
We use the email address in APIView profile. Notification system has not been changed yet to pick email address from AAD.